utilities/cli/util.py

```diff
@@ -83,16 +83,19 @@ def fatal(message: str) -> None:
 
 
 def run_command(
-    cmd: List[str], dry_run: bool = False, check: bool = True, **kwargs
+    cmd, dry_run: bool = False, check: bool = True, **kwargs
```

Comment:
How about `from typing import Union`, then:
```suggestion
    cmd: Union[str, List[str]], dry_run: bool = False, check: bool = True, **kwargs
```